libcontainer/cgroups/systemd/v2.go

```diff
@@ -302,8 +301,10 @@ func (m *unifiedManager) Destroy() error {
 	}
 
 	// XXX this is probably not needed, systemd should handle it
```

Comment:
This comment needs to be removed. Instead, maybe add that systemd vX.X.X (the one you used on CentOS 8) do not remove sub-cgroups.